Calcite always can not download when mvn install in windows env

Calcite lib always can not download when mvn install in windows env, please investigate the reason.

error log:

```
Error:  Failed to execute goal on project shardingsphere-infra-optimize: Could not resolve dependencies for project org.apache.shardingsphere:shardingsphere-infra-optimize:jar:5.0.0-RC1-SNAPSHOT: Failed to collect dependencies at org.apache.calcite:calcite-core:jar:1.26.0: Failed to read artifact descriptor for org.apache.calcite:calcite-core:jar:1.26.0: Could not transfer artifact org.apache.calcite:calcite-core:pom:1.26.0 from/to central (https://repo.maven.apache.org/maven2): Transfer failed for https://repo.maven.apache.org/maven2/org/apache/calcite/calcite-core/1.26.0/calcite-core-1.26.0.pom: Connection reset -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :shardingsphere-infra-optimize
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -26,28 +26,31 @@ on:
   pull_request:
     branches: [ master ]
 
+env:
+  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=30
+
 jobs:
-#  windows:
-#    runs-on: windows-latest
-#    steps:
-#      - name: Cache Maven Repos
-#        uses: actions/cache@v2
-#        with:
-#          path: ~/.m2/repository
-#          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-#          restore-keys: |
-#            ${{ runner.os }}-maven-
-#      - name: Support longpaths
-#        run: git config --system core.longpaths true
-#      - uses: actions/checkout@v2
-#      - name: Set up JDK 8
-#        uses: actions/setup-java@v1
-#        with:
-#          java-version: 8
-#      - name: Build with Maven
-#        run: ./mvnw -B clean cobertura:cobertura -D"maven.javadoc.skip"=true install
-#      - name: Build examples with Maven
-#        run: ./mvnw -B -f examples/pom.xml clean package -DskipTests
+  windows:
+    runs-on: windows-latest
+    steps:
+      - name: Cache Maven Repos
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Support longpaths
+        run: git config --system core.longpaths true
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build with Maven
+        run: ./mvnw -B clean cobertura:cobertura -D"maven.javadoc.skip"=true install
+      - name: Build examples with Maven
+        run: ./mvnw -B -f examples/pom.xml clean package -DskipTests
   
   unix:
     runs-on: ${{ matrix.os }}

--- a/.github/workflows/it.yml
+++ b/.github/workflows/it.yml
@@ -23,6 +23,9 @@ on:
   pull_request:
     branches: [ master ]
 
+env:
+  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=30
+
 jobs:
   MySQL:
     name: DB

